src/collective/behavior/amp/configure.zcml

```diff
@@ -7,6 +7,12 @@
 
   <five:registerPackage package="." />
 
+  <subscriber
+    for="zope.lifecycleevent.interfaces.IObjectModifiedEvent
+         zope.publisher.interfaces.IPublishTraverse"
```

Comment:
yo should register the subscriber against the behavior's marker interface and not against `IPublishTraverse` .